disnake/state.py

```diff
@@ -930,6 +931,33 @@ def parse_message_reaction_remove_emoji(
                 if reaction:
                     self.dispatch("reaction_clear_emoji", reaction)
 
+    def _handle_poll_event(
+        self, raw: RawMessagePollVoteActionEvent, event_type: Literal["add", "remove"]
+    ) -> None:
+        guild = self._get_guild(raw.guild_id)
+        answer = None
+        if guild is not None:
+            member = guild.get_member(raw.user_id)
+            message = self._get_message(raw.message_id)
+            if message is not None and message.poll is not None:
+                answer = utils.get(message.poll.answers, id=raw.answer_id)
```

Comment:
If this data is cached, it would be quite nice if the vote counts were updated (+1/-1) based on these events